### PR TITLE
probes/system_info: Better output, offline mode improvements

### PR DESCRIPTION
**Pull request by evgenyz**
Offline and normal mode output now in sync, collected data is closer to the specification and more useful.

### CODE REVIEW

File: tests/probes/system_info/test_probes_system_info.sh [hunk 1]
```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+
```
[the remark this answers is not in the document]
Comment (evgenyz) [reply tests/probes/system_info/test_probes_system_info.sh:2]: No, not really. $NAME and $VERSION are now imported from `. /etc/os-release` to accommodate the test for new data source. That's it pretty much.

File: src/OVAL/probes/independent/system_info_probe.c [hunk 2]
```diff
@@ -490,154 +485,53 @@ static FILE *_fopen_with_prefix(const char *prefix, const char *path)
...
-		saved_entry[ovec[1]] = '\0';
-		ptr = saved_entry + ovec[0];
-		int nr = atoi(ptr);
-		ret = _offline_get_menuentry(oscap_probe_root, nr);
-		pcre_free(re);
-		goto finish;
+	char *ptr = NULL;
+	int rc = pcre_exec(re, NULL, os_release_data, len, 0, 0, ovec, _REGEX_RES_VECSIZE);
+	if (rc >= 0) {
+		ptr = strndup(os_release_data+ovec[2], ovec[3]-ovec[2]);
```
[the remark this answers is not in the document]
Comment (evgenyz) [reply src/OVAL/probes/independent/system_info_probe.c:529]: Do you think that I'll explain it better than `man pcre_exec`?

[the remark this answers is not in the document]
Comment (evgenyz) [reply src/OVAL/probes/independent/system_info_probe.c:529]: Or, do you mean, describe where each part of `os_release_data` is going to be in the array?

[the remark this answers is not in the document]
Comment (jan-cerny) [reply src/OVAL/probes/independent/system_info_probe.c:529]: I meant the latter, but we can live without it.

File: tests/probes/system_info/test_probes_system_info.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (jan-cerny) [reply tests/probes/system_info/test_probes_system_info.sh:2]: Thanks.

File: src/OVAL/probes/independent/system_info_probe.c (hunk 2 above)
[the remark this answers is not in the document]
Comment (evgenyz) [reply src/OVAL/probes/independent/system_info_probe.c:531]: Added a short explanation of `ovec` elements

**Comment by jan-cerny** [comment]
The `API/XCCDF/applicability/all.sh` randomly failed.

**Comment by jan-cerny** [comment]
@openscap-ci test this please

**Comment by yuumasato** [comment]
> Works great for me on localhost and also on container images using oscap-podman. I like very much that you made it consistent.

@jan-cerny Does it mean that when scanning a host or a cointainer in the host, the hostname is the same for both cases?

**Comment by evgenyz** [comment]
The hostname is always a hostname, and it is always the name of the system being scanned (i.e. in oflline mode it'd be the contents of `$OSCAP_PROBE_ROOT/etc/hostname`).

**Pull request merged**